core/src/solve.jl

```diff
@@ -478,34 +479,76 @@ function continuous_control!(
 
         listened_node_id = listen_node_id[i]
         _, listened_node_idx = id_index(basin.node_id, listened_node_id)
+        storage_listened_basin = u.storage[listened_node_idx]
+        phi = storage_listened_basin < 10.0 ? storage_listened_basin / 10.0 : 1.0
```

Comment:
Does this need to be the same as the `reduction_factor` for the controlling node? Perhaps it's best to use the same syntax: `reduction_factor = min(s, 10.0) / 10.0`.